src/common/constants/Dependency.ts

```diff
@@ -14,6 +14,7 @@ export enum DEPENDENCY {
   CHART_MANAGER = 'chartManager',
   BREACH_MANAGER = 'breachManager',
   LOG_TABLE_MANAGER = 'logTableManager',
+  DEV_MANAGER = 'devManager',
```

Comment:
Probably a better and more detailed name for this, but all I could think of for now!